crates/re_viewer/src/ui/welcome_screen/mod.rs

```diff
@@ -78,11 +70,11 @@ impl WelcomeScreen {
             matches!(self.current_page, WelcomeScreenPage::Welcome),
             true,
         ])
-        .id_source(&self.current_page)
+        .id_source(egui::Id::new("welcome_screen_page").with(&self.current_page))
```

Comment:
Anything that implements `Hash` will work here, including:

```suggestion
        .id_source(("welcome_screen_page", &self.current_page))
```